src/libraries/System.Runtime.CompilerServices.VisualC/src/System.Runtime.CompilerServices.VisualC.csproj

```diff
@@ -7,8 +7,9 @@
   <!-- Shared CoreCLR -->
   <ItemGroup>
     <Compile Include="System\Runtime\CompilerServices\Attributes.cs" />
+    <Compile Include="System.Runtime.CompilerServices.VisualC.TypeForwards.cs" />
```

Comment:
Technically we should need this if we made the project a partial facade with the IsPartialFacade flag.

Reply:
I'll try making this a partial facade and see if it works.